rllib/algorithms/ddpg/tests/test_ddpg.py

```diff
@@ -34,10 +34,10 @@ def tearDownClass(cls) -> None:
 
     def test_ddpg_compilation(self):
         """Test whether DDPG can be built with both frameworks."""
-        config = ddpg.DDPGConfig()
+        config = ddpg.DDPGConfig().training(num_steps_sampled_before_learning_starts=0)
         config.num_workers = 0
         config.num_envs_per_worker = 2
-        config.replay_buffer_config["learning_starts"] = 0
+        config.num_steps_sampled_before_learning_starts = 0
```

Comment:
why do you have to specify this twice? you have already done this in line 37?

Reply:
**TODO**

Reply:
Fixed. Also included L38 and L39.